app/api_contract.py

```diff
@@ -0,0 +1,385 @@
+import datetime
+import enum
+import uuid
+from typing import Annotated
+
+from fastapi import FastAPI, Header, status
+from pydantic import BaseModel, EmailStr, Field, JsonValue
+
+app = FastAPI()
+
+
+JWTToken = Annotated[
+    str,
+    Header(
+        alias="Authroization",
+        title="JWT Token",
+        description="The JWT token used for authorization in `Bearer <token>` format",
+    ),
+]
+
+CloudUserToken = Annotated[
+    str,
+    Header(
+        alias="Authroization",
+        title="FinOps for Cloud user token",
+        description="The token used for authorization in `Bearer <token>` format",
+    ),
+]
```

Comment:
Similarly to the JWT token I added this to document which endpoints require JWT token, which require an OptScale user (this header) and which are public (e.g. the invitation acceptance)

However, I'm not sure how we want to provide the optscale authentication information. I went with a bearer token as it was simple to document but using the same header for different kind of tokens can be super confusing... And this all assumes we're going to use a bearer token to begin with, there are other options.

What do you think?